srtcore/epoll.cpp

```diff
@@ -73,10 +73,25 @@ modified by
 
 #include "common.h"
 #include "epoll.h"
+#include "logging.h"
 #include "udt.h"
 
 using namespace std;
 
+namespace srt_logging
+{
+extern Logger mglog;
+}
+
+using namespace srt_logging;
+
+#if ENABLE_HEAVY_LOGGING
+#define IF_HEAVY_LOGGING(instr) instr
+#define IF_DIRNAME(tested, flag, name) (tested & flag ? name : "")
+#else
+#define IF_HEAVY_LOGGING(instr) (void)0
+#endif
```

Comment:
Those macros might be helpfull in other places as well. Maybe move them to logging.h?

Reply:
Yeah, good idea. I think they might even be in use somewhere else already.

Reply:
Right. Same was in core.cpp - moving both.